eng/scripts/Language-Settings.ps1

```diff
@@ -238,19 +238,21 @@ function Update-java-CIConfig($pkgs, $ciRepo, $locationInDocRepo, $monikerId=$nu
 }
 
 # function is used to filter packages to submit to API view tool
-function Find-java-Artifacts-For-Apireview($artifactDir, $pkgName = "")
+function Find-java-Artifacts-For-Apireview($artifactDir, $pkgName)
 {
-  Write-Host "Checking for source jar in artifact path $($artifactDir)"
   # Find all source jar files in given artifact directory
-  $files = Get-ChildItem "${artifactDir}" | Where-Object -FilterScript {$_.Name.EndsWith("sources.jar")}
+  # Filter for package only in track2 data plane or management directory. Currently track1 package APIs are not reviewed/enforced by arch board"
```

Comment:
I've been trying to avoid the terms track1/track2 in our repos so lets just say filter the packages down to those in "com.azure*" groupid.